cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -89,9 +89,12 @@ func TestAddFlags(t *testing.T) {
 	}
 	f.Parse(args)
 
+	// This is unlikely to fail
```

Comment:
Sorry, still need to check the error.

Reply:
Again, this is checking the *default* value, not the user provided value.